dispatch/static/manager/src/js/components/ItemList/ItemListColumnHeaders.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import R from 'ramda'
+
+export default function ItemListColumnHeaders(props) {
+  if (!props.headers) {
+    return null
+  }
+
+  // blank first column for checkbox
+  const headers = R.insert(0, '', props.headers)
```

Comment:
I believe `const headers = [''] + props.headers` would downcast the operands into strings before concatenation, so I don't think it'd work.

Instead I'll just have the first pseudoelement be hardcoded in jsx